METRICS.md

```diff
@@ -0,0 +1,13 @@
+# Devops Group I - Metrics
+## CPU load during the last hour/the last day.
+1. Digital Ocean has great insights to the load on the cpu, which can be traced back to the last 14 days:
+https://cloud.digitalocean.com/droplets/296029286/graphs?i=d9db54&period=hour    
+2. Very insteresting fr the operations, to see if they need to increase the size of their
+## Average response time of your application's front page.
+https://tools.pingdom.com/#601961e3c0800000
```

Comment:
Also test via e.g. chrome dev tool lighthouse, such that we actually load all the resources